log/example_new_prefix_logger_test.go

```diff
@@ -0,0 +1,20 @@
+package log_test
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/go-kit/kit/log"
+)
+
+// PrefixLogger Example
+func ExampleNewPrefixLogger() {
+	var buf bytes.Buffer
+
+	logger := log.NewPrefixLogger(&buf)
+	logger.Log("question", "what is the meaning of life?", "answer", 42)
+
+	fmt.Print(&buf)
+	// Output:
+	// question=what is the meaning of life? answer=42
+}
```

Comment:
Oops looks like specs aren't passing now. I'll get that fixed up when I get home from work.